HACKING.md

```diff
@@ -42,3 +44,37 @@ For a quick compile and debug cycle, we recommend iterating code using thorough
 unit tests before going through the full workflow described above.
 
 [1]: https://wiki.qemu.org/Documentation/Networking#User_Networking_.28SLIRP.29
+
+## Containers
+
+*osbuild-composer* and *osbuild-composer-worker* can be run using Docker
+containers. Building and starting containers is generally faster than building
+RPMs and installing them in a VM, so this method is more convenient for
+developing and testing changes quickly.
+
+Each service (*composer* and *worker*) requires a configuration file and a set
+of certificates. Use the [`tools/gen-certs.sh`](./tools/gen-certs.sh) script to
+generate the certificates (using the test OpenSSL config file):
+
+    ./tools/gen-certs.sh ./test/data/x509/openssl.cnf ./containers/config  ./containers/config/ca
```

Comment:
In a follow-up this might be nice to add as a make target or something like that, just to make it even easier to get going. Or maybe I'm overthinking that :)

Reply:
Oh yeah, that would be really useful for some other things that I have in mind as well.  I'll come back to this later for sure.